src/_includes/header.html

```diff
@@ -1,4 +1,15 @@
-<header class="site-header">
+{% assign hasBanner = false %}
+{% if page.url != "/hello.html" %}
+{% assign hasBanner = true %}
+{% endif %}
+
+{% if hasBanner %}
+<div class="banner">
+  <span>Check out our booth at KubeCon San Diego for <a href="{{site.landingurl}}hello">new and upcoming features</a>!</span>
```

Comment:
I think this can be more semantic HTML:

```
<section>
<p><a></a></p>
</section>
```

Reply:
i thought section was more for when you have an article / something that would have an outline. cf

https://html.spec.whatwg.org/multipage/sections.html#the-section-element